locales/en-US/translation.json

```diff
@@ -0,0 +1,164 @@
+  "Specified users": "Specified users",
+  "Only me": "Only me",
+  "Share link of this page": "Share link of this page",
+
+  "Show latest": "Show latest",
+  "Load latest": "Load latest",
+  "was edited this page": "was edited this page.",
+
+  "Deleted Pages": "Deleted Pages",
+  "Logout": "Logout",
+
+  "page_register": {
+    "notice": {
+      "restricted": "New registration to this wiki is restricted.",
+      "restricted_defail": "In order to start using it, approval by the administrator is necessary after new registration.",
+      "google_account_continue": "Please decide user ID, name and password and continue registration."
+    },
+    "form_help": {
+      "email": "In this wiki, only the following e-mail address can be registered.",
+      "password": "Please enter at least 6 letters, numbers, alphanumeric characters, symbols etc for password.",
+      "user_id": "The user ID is used for the URL of the user page etc. Only half-width alphanumeric characters and some symbols can be used."
+    }
+  },
+
+  "page_me": {
+    "form_help": {
+      "google_connect1": "With Google Connect, you can sign in with your Google Account.",
+      "google_connect2": "Only Google Apps accounts with the following email addresses are connectable Google accounts:",
+      "google_disconnect1": "If disconnet Google, you can not login by Google Authentication.",
+      "google_disconnect2": "After disconnect, you can login using email and password."
+    }
+  },
+  "page_me_apitoken": {
+    "notice": {
+      "apitoken_issued": "API Token is not issued.",
+      "update_token1": "With updating, the new API Token will be generated.",
+      "update_token2": "You have to update the processes using the current API Token."
+    },
+    "form_help": {
+    }
+  },
+
+  "Password": "Password",
+  "Password Setting": "Password Setting",
+  "Set new Password": "Set new Password",
+  "Update Password": "Update Password",
+  "Current password": "Current password",
+  "New password": "New password",
+  "for check": "for check",
+  "Please set a password": "Please set a password",
+  "You can login email and password": "You can login <code>%s</code> and password",
+
+  "API Setting": "API Setting",
+  "API Token Setting": "API Token Setting",
+  "Current API Token": "Current API Token",
+  "Update API Token": "Update API Token",
+
+  "page_page": {
+      "notice": {
+          "version": "This is not the current version.",
+          "moved": "This page was moved from <code>%s</code>",
+          "restricted": "Access to this page is restricted"
+      }
+  },
+
+  "Rename page": "Rename page",
+  "New page name": "New page name",
+  "Current page name": "Current page name",
+  "Redirect": "Redirect",
+  "modal_rename": {
+    "help": {
+      "redirect": "Redirect to the new page if someone access <code>%s</code>"
+    }
+  },
+
+  "modal_help": {
+      "basic": {
+          "title": "基本的な機能",
+          "body1": "表示される画面には、「一覧ページ」と「ページ」の2種類があります",
```

Comment:
There are 2 types of pages: List pages (showing lists of links to other pages) and normal pages.